docs/polaris/compiling-and-linking/llvm-compilers-polaris.md

```diff
@@ -1,7 +1,7 @@
 # LLVM Compilers on Polaris
 
 This page is not about LLVM-based Cray Compiling Environment (CCE) compilers from `PrgEnv-cray` but about open source LLVM compilers.
-If LLVM compilers are needed without MPI support, simply load the `llvm` or `llvm-sycl` module. 
+If LLVM compilers are needed without MPI support, simply load the `llvm` or `oneapi` module.
```

Comment:
LLVM compilers refers to upstream LLVM. This is confusing. Simply remove `llvm-sycl` and `onepai`.

Reply:
done